packages/jumpstarter-driver-power/jumpstarter_driver_power/client.py

```diff
@@ -13,6 +14,16 @@ def on(self) -> str:
     def off(self) -> str:
         return self.call("off")
 
+    def cycle(self, quiescent_period: int = 2) -> str:
```

Comment:
```suggestion
    def cycle(self, quiescent_period: int = 2):
```

I would simply return nothing (i.e. avoid issues later if somebody is using a linter, and the linter complains because you are not capturing the output of the str... although I see we have the same on on/off hhmmmm

WDYT @NickCao ?

Reply:
> I would simply return nothing

Sure, the return value was initially added for no particular reasons. We should also make `on` and `off` return nothing.

Reply:
ack, i'll drop it

Reply:
had to change a lot of tests a result 😅

Reply:
oops, sorry for the extra noise :D we could have done it later, thanks benny!